src/app/components/my-files/my-files.component.ts

```diff
@@ -311,8 +357,14 @@ export class MyFilesComponent implements OnInit, OnDestroy {
             this.onSearchChanged();
             this.ref.detectChanges();
         }, (error) => {
-            console.error('Unable to delete file');
-            console.error('Error:', error);
+            if (error.message !== 'Http failure during parsing for http://127.0.0.1:8002/files/remove') {
```

Comment:
This should be fixed now.. If the delete is successful, there should be no error whatsoever.